Sources/NIOHTTP2/HTTP2ChannelHandler+InlineStreamMultiplexer.swift

```diff
@@ -227,20 +239,31 @@ extension NIOHTTP2Handler {
     /// `OutboundStreamOutput`. This type may be `HTTP2Frame` or changed to any other type.
     @available(macOS 10.15, iOS 13.0, watchOS 6.0, tvOS 13.0, *)
     @_spi(AsyncChannel)
-    public struct AsyncStreamMultiplexer<InboundStreamOutput> {
-        private let inlineStreamMultiplexer: InlineStreamMultiplexer
+    public struct AsyncStreamMultiplexer<InboundStreamOutput>: @unchecked Sendable {
```

Comment:
Aren't all the properties of this type `Sendable`?

Reply:
Yep, marked as `Sendable`